### PR TITLE
Updated German translations

### PR DESCRIPTION
**Pull request by theHacker**
Not sure if PRs are appreciated in this repo?
I did not want to register with https://poeditor.com/, but rather use `vim` on the `.po` files :wink:

### CODE REVIEW

**Comment by fichtner** [comment]
I can try to review and merge, but I don’t want this to be a regular workflow, ok?

Grüße
Franco

**Comment by theHacker** [comment]
I see… thank you very much :smiley:

File: de_DE.po [hunk 1]
```diff
@@ -3049,7 +3049,7 @@ msgstr "Gateway Priorität"
 #. File: /usr/core/src/www/system_gateway_groups_edit.php, line: 114
 #: 
 msgid "No gateway(s) have been selected to be used in this group"
-msgstr "Es wurden kein(e) Gateway(s) ausgewählt um in dieser Gruppe verwendet zu werden"
+msgstr "Es wurden kein(e) Gateway(s) ausgewählt, um in dieser Gruppe verwendet zu werden"
```
Comment (fichtner) [inline de_DE.po:3052]:
```suggestion
msgstr "Es wurde(n) kein(e) Gateway(s) ausgewählt, um in dieser Gruppe verwendet zu werden"
```

Ich hab den String aber mal geändert für zukünftige Versionen: https://github.com/opnsense/core/commit/35576deaa

**Comment by fichtner** [comment]
LGTM, thanks a lot! Merged.

**Pull request merged**